repose-aggregator/components/filters/saml-policy-translation-filter/src/test/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilterTest.scala

```diff
@@ -754,37 +1147,37 @@ class SamlPolicyTranslationFilterTest extends FunSpec with BeforeAndAfterEach wi
     }
 
     it("should initialize the cache when given a config") {
-      ReflectionTestUtils.getField(filter, "cache") shouldBe null
+      ReflectionTestUtils.getField(filter, "org$openrepose$filters$samlpolicy$SamlPolicyTranslationFilter$$policyCache") shouldBe null
```

Comment:
It's getting this name now because `policyCache` is being used in an anonymous function (what we're passing in to `.foreach(policyCache.invalidate(_))` in the onNewAtomEntry method) which is getting converted into the inner class:
```
public final INNERCLASS org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilter$$anonfun$onNewAtomEntry$2
```

If you comment out that line, the field name goes back to normal.

Of note, `feedId` also has the long name now, too, because it's in a foreach in the doDestroy method.  I mention this because you made the comparison between the two at some point.